### ISSUE TITLE
the output is empty

### ISSUE THREAD

**Issue by chris111110** [yangheng95/PyABSA#275]
hi, when I try to extract the aspect item by using  the aspect_extractor.predict, but the output is empty, is there any limitation for the length of the text?

**Comment by yangheng95** [comment]
Yes. Them max sequence length is set to 128 generally, which improves the efficiency. If you want to have better performance, try to train on your own dataset and set larger max length.

**Comment by chen-bowen** [comment]
Could you let us know where this max_length parameter is set? Any way to override it?

**Comment by yangheng95** [comment]
In the config object, e.g., set config.max_seq_length=512
